packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3091,11 +3117,16 @@ const waitForSeq = async (
     // TODO: remove cast to any when actual event is determined
     deltaManager.on("closed" as any, reject);
```

Comment:
@kian-thompson - FYI, as you work on dispose/close on Container, we should have guidance for developers how to change their code, and ideally scan repo and fix our usage, if usage needs to be changed. I think there is no changes, but let's double check :)